source/common/network/address_impl.h

```diff
@@ -236,6 +291,12 @@ class PipeInstance : public InstanceBase {
   }
 
 private:
+  /**
+   * Construct from an existing unix address.
```

Comment:
add ", returning error status without throwing"

And modify the description of the other constructors (here and or ipv4/ipv6) to indicate they can throw.

Reply:
Thanks, will add comments.